test/unit/controllers/ChoiceControllerSpec.scala

```diff
@@ -23,18 +23,24 @@ import models.{DeclarationStatus, ExportsDeclaration}
 import org.mockito.ArgumentMatchers.any
 import org.mockito.Mockito.when
 import play.api.libs.json.{JsValue, Json}
+import play.api.mvc.{AnyContentAsJson, Request}
+import play.api.test.FakeRequest
 import play.api.test.Helpers._
 import unit.base.ControllerSpec
 import unit.mock.ErrorHandlerMocks
 import views.html.choice_page
+import utils.FakeRequestCSRFSupport._
 
 import scala.concurrent.Future
 
 class ChoiceControllerSpec extends ControllerSpec with ErrorHandlerMocks {
   import ChoiceControllerSpec._
 
-  private val existingDeclaration =
-    aDeclaration(withId("declarationId"), withChoice(Choice.AllowedChoiceValues.SupplementaryDec))
+  private def existingDeclaration(choice: String = Choice.AllowedChoiceValues.SupplementaryDec) =
```

Comment:
I know it's old code, but can you change import from just Choice to `Choice.AllowedChoiceValues._`?

Usage will be shorter and more redable